tracing/zipkin/src/main/java/module-info.java

```diff
@@ -25,13 +25,13 @@
         in = {HelidonFlavor.MP, HelidonFlavor.SE, HelidonFlavor.NIMA},
         path = {"Tracing", "Zipkin"}
 )
-module io.helidon.tracing.zipkin {
+module io.helidon.tracing.providers.zipkin {
```

Comment:
This file should not exist, it should be in providers/zipkin.

Reply:
Fixed